best-practices/README.md

```diff
@@ -105,6 +107,7 @@ Rails
 [redirects]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.30
 [Spring binstubs]: https://github.com/sstephenson/rbenv/wiki/Understanding-binstubs
 [prevent tampering]: http://blog.bigbinary.com/2013/03/19/cookies-on-rails.html
+[class constant in association]: https://github.com/thoughtbot/guides/blob/master/style/rails/sample.rb
```

Comment:
A problem I've noticed is that we link to a line on master, but that line could move around at any point. Linking to an file as of a specific commit isn't possible unless we do two commits, since the act of commit doesn't exist yet. I think this is small enough to leave it as is, as I don't like letting the line numbers potentially drift over time.